asmcli/lib/validate.sh

```diff
@@ -0,0 +1,153 @@
+x_validate_install_args() {
```

Comment:
Question: does it make more sense to you to separate the experimental-only lib functions from the rest of them? I don't have a strong opinion yet.

Reply:
Synced offline to have separate parse and validation for the experimental commands.